teamcity/conf.yaml.example

```diff
@@ -28,11 +28,12 @@ instances:
     # rather than just that a successful build happened
     # is_deployment: true
 
+    # Optional, this allows you to connect to your server through HTTPS. Defaults to False.
+    # use_https: false
```

Comment:
If we think this is too complex we could build the URLs like this:
```
LAST_BUILD_URL = "{server}/guestAuth/app/rest/builds/"
```
prepending `http://` when the protocol is not contained in `server` for backward compat. New users should set `server` so that it also has a protocol prepended (http or https depending on use case).

Reply:
Yes, I think this is the better option.